README.md

```diff
@@ -49,6 +49,14 @@ DockerClient client = new DockerClientConfiguration(new Uri("npipe://./pipe/dock
      .CreateClient();
 ```
 
+or to connect to your local [Docker for Mac](https://docs.docker.com/docker-for-mac/) deamon using Unix sockets:

```

Comment:
Small typo. It should meant _daemon_.

Reply:
Fixed, also for the Windows version, from which I copy/pasted the text. 😉